app/models/nccn_guideline.rb

```diff
@@ -0,0 +1,3 @@
+class NccnGuideline < ActiveRecord::Base
+  has_many :assertions
```

Comment:
Do we need to add the corresponding relation in `Assertion`?

Reply:
It's there: https://github.com/griffithlab/civic-server/blob/issue_1255/app/models/assertion.rb#L15